db/migrations/02_create_spaces_table.sql

```diff
@@ -1 +1,7 @@
-CREATE TABLE spaces(id SERIAL PRIMARY KEY, username VARCHAR(50), description VARCHAR(300), price INT);
+CREATE TABLE spaces(
+  id SERIAL PRIMARY KEY,
```

Comment:
Thinking about it, maybe we would be better off having the id in the spaces table be space_id, and the id in the users table user_id just so we don't have 2 "id" columns that mean different things?

Reply:
thanks Stu, I have made a new commit which takes into account your comments